export const handleErrorGraciously = () => {
  return new Error('An error occurred.')
}
